### ISSUE TITLE
Favourite Manga doesn't show custom cover in browse and search.

### ISSUE THREAD

**Issue by FourTOne5** [tachiyomiorg/tachiyomi#5489]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/tag/v0.11.1)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

0.11.1-3537

### Android version

6.0

### Device

LAVA iris 820

### Steps to reproduce

Add any manga and set a custom cover.

![In Libary](https://user-images.githubusercontent.com/59261191/124227804-10650800-db2d-11eb-8dc2-7e12a5189ced.png)

Search for the same manga.

### Expected behavior

![Before #5440](https://user-images.githubusercontent.com/59261191/124228151-91240400-db2d-11eb-9bef-53defd9c319c.jpg)

### Actual behavior

![After #5440](https://user-images.githubusercontent.com/59261191/124228459-0db6e280-db2e-11eb-8b5d-fd159ef06e4b.png)

### Other details

Started after commit #5440

**Comment by vengiare** [comment]
I'd say it's intentional. You're viewing the source and not your library so it's just appropriate to show the cover from the source.

**Comment by FourTOne5** [comment]
If it's intentional they'll just close the issue.

**Comment by arkon** [comment]
Don't think it was a conscious choice, but does seem like the proper thing to do.
